pkgs/top-level/all-packages.nix

```diff
@@ -27444,10 +27444,7 @@ in
   wrapNeovimUnstable = callPackage ../applications/editors/neovim/wrapper.nix { };
   wrapNeovim = neovim-unwrapped: lib.makeOverridable (neovimUtils.legacyWrapper neovim-unwrapped);
   neovim-unwrapped = callPackage ../applications/editors/neovim {
-    lua =
-      # neovim doesn't work with luajit on aarch64: https://github.com/neovim/neovim/issues/7879
-      if stdenv.isAarch64 then lua5_1 else
-      luajit;
+    lua = luajit;
```

Comment:
Have you verified that luajit now works on aarch64?  Reading through the neovim issue, there's still people running into this bug.  Apparently it might work on some unspecified master version of luajit, but I don't know if that version is already in nixpkgs.

Reply:
https://github.com/NixOS/nixpkgs/pull/110837#discussion_r564673401 @thiagokokada ?

Reply:
We could potentially extract this into another PR if there are concerns, I did manage to run this on m1 mac earlier without issues, and I rely on lua heavily for configuration. I'd guess that should mean it works?

Reply:
Ah, if you've tested it then that should be fine.  I just wanted to make sure that you didn't remove the override just because the neovim bug has been closed.